war/package.json

```diff
@@ -15,9 +15,12 @@
     "build": "yarn prod",
     "start": "yarn dev --watch",
     "test": "jest --ci --config=jest.config.js",
-    "lint:js": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
-    "lint:css": "stylelint src/main/less --custom-formatter node_modules/stylelint-checkstyle-reporter/index.js -o target/stylelint-warnings.xml",
+    "lint:js": "eslint . --ext js",
+    "lint:js-ci": "eslint . --ext js -f checkstyle -o target/eslint-warnings.xml",
+    "lint:css": "stylelint src/main/less",
+    "lint:css-ci": "stylelint src/main/less --custom-formatter stylelint-formatter.js -o target/stylelint-warnings.xml",
     "lint:css:fix": "stylelint src/main/less --fix",
+    "lint:ci": "yarn lint:js-ci && yarn lint:css-ci",
```

Comment:
extracted from https://github.com/jenkinsci/jenkins/pull/6863/.

difficult to develop locally without as you don't know what it's complaining about, in the case of eslint it just silently exited with 1 before